### DISCUSSION TITLE
Using version numbers in your project names

### DISCUSSION THREAD

**Discussion by cpyburn78** [NVIDIA-Omniverse/PhysX#46] [Ideas]
Big name projects have quit using version numbers in there projects because it is bad practice and quite annoying to the customers to either rename all the folders to fit their custom folder naming structure or to go update every dependency location.  Would be super nice if you all could adapt naming PhysxX.X.X.X to just Physx with no versions.  See screen shot.
![image](https://user-images.githubusercontent.com/118981685/203784014-662866ac-102c-455d-871a-c5e3baebb6f5.png)

**Comment by preist-nvidia** [comment]
I'm not sure I follow - if you get the repo, there are no version numbers in the path to the include folder, unless you clone yourself into a versioned folder.

Edit: Ah, I assume you mean projects that use PhysX, so this is addressed to users.
